static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -1919,54 +1934,62 @@ export class TraceTree extends TraceTreeEventDispatcher {
 
     let traceStats: {
       javascriptRootTransactions: Array<TraceTree.Transaction | TraceTree.EAPSpan>;
-      orphans: number;
+      orphan_errors: TraceTree.TraceErrorIssue[];
+      orphan_spans: number;
```

Comment:
minor nit: Feels weird to have one type be an array and another type be a number. I'd update `orphan_errors` to a `number` if we only need the length.

Reply:
Yeah I only need the length here, I'll update all the attrs to *_count.